tests/test_armv7cpu.py

```diff
@@ -986,6 +986,86 @@ def test_bl_neg(self):
         self.assertEqual(self.rf.read('PC'), pre_pc - 4)
         self.assertEqual(self.rf.read('LR'), pre_pc + 4)
 
+    # CBZ/CBNZ
+
+    @itest_setregs("R0=0")
+    @itest_custom("cbz r0, 0x2a", mode=CS_MODE_THUMB)
+    def test_cbz_taken(self):
+        pre_pc = self.rf.read('PC')
+        self.cpu.execute()
+        self.assertEqual(self.rf.read('PC'), pre_pc + 42)
+
+    @itest_setregs("R0=1")
+    @itest_custom("cbz r0, #0x2a", mode=CS_MODE_THUMB)
```

Comment:
Is there any reason why do we use sometimes `0x2a` and sometimes `#0x2a`?

Reply:
Based on [this other StackOverflow answer](https://stackoverflow.com/a/51987780), I agree that it's better to use `#` to keep it consistent. I have made the necessary changes in my latest commit.